### PR TITLE
Build AmberTools 22

### LINKED ISSUE #81: We need to work on creating a conda package for AmberTools22, which will be released soon
### Comment:

The release of AmberTools22 should be ready later this month (April, 2022).  We have release candidates ready, but I'd like to find out what problems (if any) we will find in updating the current conda packages.  Here is the updated URL

   https://ambermd.org/downloads/AmberTools22.02apr22.tar.bz2

In the ideal world, one could substitute the new URL for the existing one in the `meta.yaml` file.  I don't see any obvious problems in `build.sh`, but the code changes are pretty big, so something is likely to break....For sure, the three patch files invoked after downloading the source are unlikely to apply cleanly.

Unfortunately, I am not much of a conda or conda-forge person.  I edited build.sh to skip the `update_amber` step, and tried `conda build recipe`.  Pretty early on I encountered a version problem with cpptraj:
```
+ rm /Users/case/conda-bld/ambertools_1649863164081/work/AmberTools/src/cpptraj/src/xdrfile/version
rm: /Users/case/conda-bld/ambertools_1649863164081/work/AmberTools/src/cpptraj/src/xdrfile/version: No such file or directory
```
I could try to track this down, but it seems likely that someone (@jaimergp  @simonbray ?) would be much more likely to know what to do here, and in whatever pops up next.  But let me know how I can help, if needed.

...thanks!....dac

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Closes #81

### CODE REVIEW

**Comment by jaimergp** [comment]
@dacase Trying out the RC here!

**Comment by dacase** [comment]
Update: there is a fourth (final?) release candidate here:

   https://ambermd.org/downloads/AmberTools22.15apr22.tar.bz2

From the point of view of conda, there shouldn't (!?) be any changes.  Let me know if there is something I can do to help check things out -- I could do something like `conda install -c xxxxx ambertools=22`  and look for problems that might go beyond what the test suite itself checks.

Thanks again for working on this!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://ambermd.org/downloads/AmberTools21jlmrcc.tar.bz2
-  sha256: f55fa930598d5a8e9749e8a22d1f25cab7fcf911d98570e35365dd7f262aaafd
+  url: https://ambermd.org/downloads/AmberTools22.02apr22.tar.bz2
+  sha256: b2050e495f23835aea12cec030023cdef63b198a67fa4c0661ac9a715462a210
```
Comment (jaimergp) [inline recipe/meta.yaml:12]:
```suggestion
  url: https://ambermd.org/downloads/AmberTools22.15apr22.tar.bz2
  sha256: c9933fa62c67e43e714cd6dcdc7aa69d156c89f5f92543a1fb8e7b4ecc916cb1
```

**Comment by jaimergp** [comment]
@dacase - which ParmEd version are you bundling now? We need to update this line:


https://github.com/jaimergp/ambertools-feedstock/blob/57dfbeefec5c44e80f13d8ecc72892c2d59ccc32/recipe/meta.yaml#L67

**Comment by dacase** [comment]
parmed in AmberTools22 still says version 3.4.1.  I'll ping Jason Swails to see if we are doing something wrong.

**Comment by jaimergp** [comment]
I mean, I was assuming you had updated it. If that didn't happen we are good!

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by jaimergp** [comment]
Excellent.

One more thing, I am seeing some segfaults on the macos builds. Can you take a look at [the logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=494120&view=logs&j=4b3757f9-99c5-5891-6c0c-93d7030cdf7d&t=2ff5d027-ebfd-5364-5607-51694385a7af&l=41553)?

For example:

```
8667-export TESTsander='/Users/runner/amber/bin/sander'; cd gact_ips && ./Run.ips_sgmdbd3
8668-
8669-Program received signal SIGSEGV: Segmentation fault - invalid memory reference.
8670-
8671:Backtrace for this error:
8672-#0  0x121faa4dc
8673-#1  0x121fa984d
8674-#2  0x7fff6d3de5fc
8675-#3  0x10d38b9fd
8676-#4  0x10d38c816
8677-#5  0x10d50db74
8678-#6  0x10d50598c
8679-#7  0x10d5059eb
8680-Segmentation fault
```

**Comment by jaimergp** [comment]
Btw, here are the [artifacts](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=494120&view=artifacts&pathAsName=false&type=publishedArtifacts).

**Comment by dacase** [comment]
I've intermittently seen these `gact_ips` segfaults before.  We can probably track them down, but I'm actually not too worried here.
I bet that literally no one uses this option, and if they do get a segfault, they will know something is wrong, and we may need to bump up the problem priority.

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by dacase** [comment]
Please see updated comments in #81

**Comment by jaimergp** [comment]
Package previews available with this command, @dacase 

```
$ conda create -n ambertools-22 -c jaimergp/label/ambertools-22 -c conda-forge ambertools=22
```

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Pull request merged**